### ISSUE TITLE
Return an empty result 

### ISSUE THREAD

**Issue by novikov-an** [nawa/node-kinopoisk-ru#4]
I'm using this demo: https://npm.runkit.com/kinopoisk-ru and by posting default code it returns an empty string. Is this Library still working?

**Comment by nawa** [comment]
No, the library is not maintained. It is based on html code of kinopoisk pages and they have been changed after 2 years. Login is broken I think and  film parser too
Project contains integration tests and you can try to update the library
